docs/install_apt_debian.md

```diff
@@ -4,10 +4,11 @@ title: "Installation: Debian"
 
 # Debian Installation
 
-Caffe packages are available for `Debian/unstable`. Debian/stable users
-should take a look at Ubuntu installation instruction.  
+Caffe packages are available for `Debian/unstable` (Debian 10). Debian/stable
+(jessie) and Debian/testing (stretch) users should take a look at [Ubuntu
```

Comment:
Currently Debian/testing users should be able to install these caffe packages. With a sources.list like this and a configured `apt_preferences`(man apt_preferences) :
```
deb http://mirror/debian testing main contrib non-free
deb http://mirror/debian unstable main contrib non-free
```
I didn't refer this because this is advanced usage, and it may break after several months.

There is still hope that Caffe can enter Debian/Stretch, if some warm-hearted volunteer can fix the skimage package.

https://tracker.debian.org/pkg/caffe